scan-payment/src/main/java/com/getbouncer/scan/payment/analyzer/PaymentCardOcrState.kt

```diff
@@ -0,0 +1,6 @@
+package com.getbouncer.scan.payment.analyzer
+
+data class PaymentCardOcrState(
```

Comment:
This can probably be moved into `PaymentCardOcrAnalyzer` if we separate `SSDObjectDetect` from `NameDetectAnalyzer`